gallery/gallery/lib/data/demos.dart

```diff
@@ -541,6 +542,21 @@ List<GalleryDemo> materialDemos(BuildContext context) {
         ),
       ],
     ),
+    GalleryDemo(
+      title: GalleryLocalizations.of(context).demoTooltipTitle,
+      icon: GalleryIcons.tooltip,
+      subtitle: GalleryLocalizations.of(context).demoTooltipSubtitle,
+      configurations: [
+        GalleryDemoConfiguration(
```

Comment:
final var at top?

Reply:
Good Idea, I'll do that in a followup PR though, since it'll modify a bunch of this file